Text_Mining.py

```diff
@@ -0,0 +1,161 @@
+import requests
+import pickle
+import os.path
+import nltk
+
+
+"""Downloads 'Alice in Wonderland'
+"""
+
+# Save data to a file (will be part of your data fetching script)
+if os.path.isfile('alice_in_wonderland.pickle') == False:
+    alice_in_wonderland_full_text = requests.get('http://www.gutenberg.org/files/11/11-0.txt').text
+    f = open('alice_in_wonderland.pickle', 'wb')
+    pickle.dump(alice_in_wonderland_full_text, f)
+    f.close()
+else:
+    # Load data from a file (will be part of your data processing script)
+    input_file = open('alice_in_wonderland.pickle', 'rb')
+    reloaded_copy_of_texts1 = pickle.load(input_file)
+
+
+"""Downloads 'Tom Sawyer'"""
+
+if os.path.isfile('tom_sawyer.pickle') == False:
+    Tom_Sawyer_full_text = requests.get('http://www.gutenberg.org/files/74/74-0.txt').text
+    f2 = open('tom_sawyer.pickle', 'wb')
+    pickle.dump(Tom_Sawyer_full_text, f)
+    f.close()
+else:
+    # Load data from a file (will be part of your data processing script)
+    input_file2 = open('tom_sawyer.pickle', 'rb')
+    reloaded_copy_of_texts2 = pickle.load(input_file2)
+
+"""Cuts out beginning and end sections from each book"""
+
+def cut_alice(text):
+    index = text.find('***')
+    new_text = text[index+3:]
+    index2 = new_text.find('***')
```

Comment:
An in-line comment describing what you are doing here would be helpful for readers to understand your code. I am assuming that a project Gutenberg files has some header chunk that starts and ends with `***` and you are trying to look for it here. If someone is not familiar with how Gutenberg files are structured, they will have hard time trying to understand what this code does. Therefore, this would be good place for some in-line documentation.